diff --git a/openlane/scripts/pyosys/synthesize.py b/openlane/scripts/pyosys/synthesize.py
index 317f09f..5d14404 100644
--- a/openlane/scripts/pyosys/synthesize.py
+++ b/openlane/scripts/pyosys/synthesize.py
@@ -309,7 +309,7 @@ def synthesize(
     if config["SYNTH_ELABORATE_ONLY"]:
         openlane_proc(d, report_dir)
         if config["SYNTH_ELABORATE_FLATTEN"]:
-            d.run_pass("flatten")
+            d.run_pass("flatten", "-noscopeinfo")
         d.run_pass("setattr", "-set", "keep", "1")
         d.run_pass("splitnets")
         d.run_pass("opt_clean", "-purge")
